src/subscription.js

```diff
@@ -60,6 +60,10 @@ var Subscriber = require('./subscriber.js');
  * @param {string} name The name of the subscription.
  * @param {object} [options] See a
  *     [Subscription resource](https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.subscriptions)
+ * @param {object} [options.batching] Batch configurations for sending out
+ *     Acknowledge and ModifyAckDeadline requests.
+ * @param {number} [options.batching.maxMilliseconds] The maximum amount of time
```

Comment:
I think we can put that default value like this: `[options.batching.maxMilliseconds=100]`.

Reply:
That was my initial thought too, but it appears we've been documenting default values the old way in PubSub so I went this way for the sake of consistency.